ui/src/app/store/space/slice.ts

```diff
@@ -0,0 +1,16 @@
+import { SliceCaseReducers } from "@reduxjs/toolkit";
+
+import { generateSlice, GenericSlice } from "../utils";
+import { Space, SpaceState } from "./types";
+
+type SpaceReducers = SliceCaseReducers<SpaceState>;
+
+export type SpaceSlice = GenericSlice<SpaceState, Space, SpaceReducers>;
+
+const SpaceSlice = generateSlice<Space, SpaceState["errors"], SpaceReducers>({
```

Comment:
I saw some warnings in another slice where there was a type and slice with the same name (in this case there is `type SpaceSlice` and `const SpaceSlice`). But I now can't figure out how to reproduce it...